hubspot3/tickets.py

```diff
@@ -18,6 +18,19 @@ class TicketsClient(BaseClient):
 
     default_batch_properties = ["subject"]
 
+    """
+    Since this value is not defined by hubspot, and the tickets API is
+    different from the others I won't assume that this number is equal
+    for all APIs. Waiting on an anwser from Hubspot for a more precise
+    value
+    """
+    _maximum_request_length = 15500
```

Comment:
You should use upper case for these two constants, especially since they are default parameters for functions defined below (the value for the default parameter is this value as defined when the class is parsed by the interpreter, changing this value later will not change the default parameter value).

If you intended to make the parameter value change when this value is changed, you should use the value `None` for the default parameter, and check

```
if max_properties_request_length is None:
    max_properties_request_length = self._maximum_request_length
```

(Same thing applies to the other constant)